app/models/jobseeker_profile.rb

```diff
@@ -70,6 +70,13 @@ def replace_employments!(new_employments)
     end
   end
 
+  def replace_training_and_cpds(new_training_and_cpds)
```

Comment:
```suggestion
  def replace_training_and_cpds!(new_training_and_cpds)
```

Reply:
Ah yeah, good call. Will make that change!

Yeah it's used here! https://github.com/DFE-Digital/teaching-vacancies/pull/6835/files#diff-fa61ca8687a1914d46866fd3c81eddf807b27cb0076442bdf1b96d03100966f7R109

Reply:
Thanks for checking!